.gitignore

```diff
@@ -1,3 +1,4 @@
 *.pyc
 env/
 .DS_Store
+.idea
```

Comment:
Nitpick: arguably this should be in your personal global `.gitignore` rather than in a project-specific one. Not a blocker.